docs/concepts/policy/pod-security-policy.md

```diff
@@ -124,7 +126,21 @@ configMap, downwardAPI, emptyDir, persistentVolumeClaim, secret, and projected.
 
 ### Host Network
  - *HostPorts*, default `empty`. List of `HostPortRange`, defined by `min`(inclusive) and `max`(inclusive), which define the allowed host ports.
- 
+
+### AllowPrivilegeEscalation
+
+Gates whether or not a user is allowed to set the security context of a container
+to `allowPrivilegeEscalation=true`. This field defaults to `false`.
+
+### DefaultAllowPrivilegeEscalation
+
+Sets the default for the security context `AllowPrivilegeEscalation` of a container.
+This bool directly controls if the `no_new_privs` flag will be set on the
+container process. It defaults to `nil`. The default behavior of `nil` will
+allow privilege escalation so as to not break setuid binaries. Setting it to `false`
+will ensure that no child process of a container can gain more privileges than
+it's parent.
+
```

Comment:
controls if -> controls whether

will be set -> gets set

will allow privilege escalation -> allows privilege escalation

will ensure -> ensures

it's -> its